app/src/main/java/com/dessalines/thumbkey/db/AppDb.kt

```diff
@@ -50,6 +50,9 @@ const val DEFAULT_SPACEBAR_MULTITAPS = 1
 const val DEFAULT_SLIDE_SENSITIVITY = 9
 const val DEFAULT_SLIDE_ENABLED = 0
 const val DEFAULT_BACKDROP_ENABLED = 0
+const val DEFAULT_KEY_PADDING = 0
+const val DEFAULT_KEY_BORDER_WIDTH = 0
```

Comment:
Make this default to `1`, otherwise it doesn't show any borders, and doesn't look to great by default.